bciers/libs/utils/src/enums.ts

```diff
@@ -73,9 +73,15 @@ export enum Status {
   APPROVED = "Approved",
   DECLINED = "Declined",
   PENDING = "Pending",
-  DRAFT = "Draft",
   CHANGES_REQUESTED = "Changes Requested",
   NOT_STARTED = "Not Started",
+  DRAFT = "Draft",
```

Comment:
I tried to clean the statuses up but they're still used in a couple places (mainly reg1) and the build breaks. We can remove them as part of the next cleanup sweep